content/posts/2023-05-15-mac-learning.md

```diff
@@ -0,0 +1,303 @@
+---
+title: "MAC Learning Basics"
+date: 2023-05-15 09:03:20 +0200
+draft: false
+---
+
+This post shows illustrates _dynamic learning of unicast MAC addresses_
```

Comment:
> "shows illustrates"

Seems redundant?  Either "This post shows ..." or "This post illustrates", right?

Reply:
Thanks.